src/recommendationservice/client.py

```diff
@@ -24,18 +23,13 @@
 logger = getJSONLogger('recommendationservice-server')
 
 if __name__ == "__main__":
-    # get port
-    if len(sys.argv) > 1:
-        port = sys.argv[1]
-    else:
-        port = "8080"
-
     # set up server stub
-    channel = grpc.insecure_channel('localhost:'+port)
-    
+    channel = grpc.insecure_channel('localhost:8080')
```

Comment:
Maybe we should consider using the env var `RECOMMENDATION_SERVICE_PORT` here as well.

Reply:
That would be better. I could load the port or address from the `.env` file using a library such as https://pypi.org/project/python-dotenv/